doc/zstd_compression_format.md

```diff
@@ -478,7 +477,7 @@ For values spanning several bytes, convention is little-endian.
 
 __`Size_Format` for `Raw_Literals_Block` and `RLE_Literals_Block`__ :
 
-- Value x0 : `Regenerated_Size` uses 5 bits (0-31).
+- Value X0 : `Size_Format` uses 1 bit, `Regenerated_Size` uses 5 bits (0-31).
```

Comment:
`Size_Format` should be repeated on all 3 use cases to reduce confusion.

Additional minor details : 
- nowadays, I would rather write `Value ?0` rather than `Value X0`
- Let's properly align those lines in text mode. 
  This file is supposed to be read through a markdown parser, but it should remain easy and agreable to read in pure text mode too. That's the strength of the markdown format.